qutebrowser/config/configdata.yml

```diff
@@ -88,6 +88,14 @@ session_default_name:
 
     If this is set to null, the session which was last loaded is saved.
 
+session_lazy_restore:
```

Comment:
Like you say in your original comment, it'd make sense to name this `session.lazy_restore` instead.

You should be able to rename `session_default_name` to `session.default_name` by changing the name above, adding a `renamed:` block (see others in this file), and adjusting all code using `session_default_name`.